PowerPointLabs/PowerPointLabs/CropLab/CropLabSettings.cs

```diff
@@ -0,0 +1,26 @@
+﻿namespace PowerPointLabs.CropLab
+{
+    public enum AnchorPosition
+    {
+        TopLeft,
+        Top,
+        TopRight,
+        MiddleLeft,
+        Middle,
+        MiddleRight,
+        BottomLeft,
+        Bottom,
+        BottomRight
+    }
+
+    internal static class CropLabSettings
+    {
+        private static AnchorPosition anchorPosition = AnchorPosition.Middle;
```

Comment:
Why are there 2 vars anchorPostition and AnchorPosition? Try `public static AnchorPosition AnchorPosition { get; set; } = AnchorPosition.Middle;`